test/csv/interface/test_delegation.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: false
+
+require_relative "../helper"
+
+class TestCSVInterfaceRead < Test::Unit::TestCase
+  def test_stringio_missing_methods_delegation
+    csv = CSV.new("h1,h2")
+
+    assert_equal(nil, csv.binmode?)
+    assert_equal(nil, csv.flock(0))
+    assert_equal(nil, csv.ioctl(0, 0))
```

Comment:
I think that `NotImplementedError` is suitable instead of returning `nil`.
Because `IO#ioctl` raises `NotImplementedError` on environment that doesn't have `ioctl(2)` such as Windows.

Reply:
Good catch! Got the system calls to raise `NotImplementedError` in `StringIO` and the likes.